flw/src/main/resources/flw.properties

```diff
@@ -0,0 +1,7 @@
+#what time of day should the process of generating the targetFile start
+flw.purge_invalid_flw_start_time=06:00
+
+#one day = 1000ms x 60s x 60m x 24h = 86400000
+flw.purge_invalid_flw_ms_interval=86400000
```

Comment:
I'm uncomfortable specifying a day interval in seconds (I do the same in the IMI module)
It feels like we should be able to explicitly schedule an event daily at a specific hour.
Opinion?